README.md

```diff
@@ -43,7 +43,7 @@ This command generates static content into the `build` directory and can be serv
 Recommended to read the following part before you start to contribute the docs.
 
 - Chinese docs is under `/docs-zh`
-- VitePress markdown features [VitePress markdown](https://vitepress.vuejs.org/guide/markdown.html)
+- VitePress Markdown features [VitePress markdown](https://vitepress.vuejs.org/guide/markdown.html)
```

Comment:
```suggestion
- VitePress Markdown features [VitePress Markdown](https://vitepress.vuejs.org/guide/markdown.html)
```

I guess the tool doesn't check the text inside the square brackets.

Reply:
Maybe. I'll check it.

Reply:
I have confirmed that the tool will check the text in square brackets.

Currently, the spell check of this tool is still experimental. Many problems need to be solved.

This line's situation is a bug. I'll open an issue for it.